functions/handlers/books.js

```diff
@@ -17,7 +17,7 @@ exports.getAllBooks = (req, res) => {
           commentCount: doc.data().commentCount,
           favCount: doc.data().favCount,
           chapterCount: doc.data().chapterCount,
-          userImage: doc.data().userImage,
```

Comment:
Hold up. We still need 

`userImage : doc.data().userImage `

If I'm not mistaken, when we display the book infromation, we still need to display the user's profile picture right? if we take it out here we probably can't do that !

Reply:
In my opinion, we don't need to display user's profile picture because books which belong to user are displayed in the content creation page, /publish/username. What do you think about it?

Reply:
Yes but if you're a user and you're looking at someone else's book, how are you going to display that person's picture?

Reply:
> Yes but if you're a user and you're looking at someone else's book, how are you going to display that person's picture?
Oh, I see. we need book of image and person's picture.